site/docs/intro.md

```diff
@@ -9,7 +9,7 @@ Today, we're going to break your old testing habits and learn some new ones.
 This workshop consists of several sections, each focusing on a different aspect of improving your testing practices.
 
 1. [Bad habits](./category/bad-habits/) - We'll start by exploring some common but outdated testing patterns, and ways to improve them.
-1. [Old habits](./category/old-habits/) - We'll start by exploring some common but outdated testing patterns, and ways to improve them.
+1. [Old habits](./category/old-habits/) - After that, we'll look at legacy testing frameworks and libraries that are still in use today.
```

Comment:
This was a duplicate of the first one - so rephrased it a bit so it's not the same.